### PR TITLE
Fixed runtime issue.

### PR DESCRIPTION
**Pull request by helena**
I just created src under package 'fake' to see logging in SBT run, just fyi.

### CODE REVIEW

File: src/main/scala/fake/FakeMessage.scala [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package fake
+
+import java.util.Date
+
+/**
+ * Created by cwheeler on 5/18/15.
+ */
+case class FakeMessage(messageId: String, messageContent: String, timestamp: Date) extends Serializable
```
Comment (helena) [inline src/main/scala/fake/FakeMessage.scala:8]: Cleaned up your 'class' and made it a scala case class with the appropriate 'apply' function

File: src/main/scala/fake/FakeDataStreamer.scala [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  def generateReceiverStream(ssc: StreamingContext): Unit = {
+
+    val customReceiverStream = ssc.receiverStream[String](new FakeDataReceiver())
+
+    val vsStream = customReceiverStream.filter(_.nonEmpty).map(FakeMessage(_))
+
+    // your life would be much easier if you didn't camelCase your table field names in cql but instead just did
+    // message_id, message_content, then you would not need to specify SomeColumns with fieldname as x etc.
+    // That said, I did create a ticket for us to handle those that do, so you don't have to specify 'as' :-)
+    val columnsForCamelCaseAlias = SomeColumns("messageId" as "messageId", "messageContent" as "messageContent", "timestamp" as "timestamp")
```
Comment (helena) [inline src/main/scala/fake/FakeDataStreamer.scala:39]: This is the main issue sadly

File: build.sbt [hunk 3]
```diff
@@ -1,8 +1,9 @@
-name := "cassandra-connector-failure-demo"
 
-version := "1.0"
+name := "cassandra-connector-success-demo"
 
-scalaVersion := "2.10.4"
+version := "1.1"
+
+scalaVersion := "2.10.5"
```
Comment (helena) [inline build.sbt:6]: this helped, it's the version of scala we build 1.2.1 against.

File: launch.sh [hunk 4]
```diff
@@ -1 +1 @@
-spark-submit --master local[6] target/scala-2.10/cassandra-connector-failure-demo-assembly-1.0.jar
+spark-submit --master local[6] target/scala-2.10/cassandra-connector-success-demo-assembly-1.1.jar
```
Comment (helena) [inline launch.sh:1]: That's right, I went there...

File: src/main/resources/log4j.properties [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (helena) [inline src/main/resources/log4j.properties:1]: Just added for SBT run to see logging aside from the log hailstorm in spark-submit

**Comment by lostinplace** [comment]
thanks for the pull request, and for cleaning up my scala.

**Pull request merged**